nicegui/elements/input.py

```diff
@@ -16,7 +16,7 @@ def __init__(self,
                  password_toggle_button: bool = False,
                  on_change: Optional[Callable[..., Any]] = None,
                  autocomplete: Optional[List[str]] = None,
-                 validation: Dict[str, Callable[..., bool]] = {}) -> None:
+                 validation: Optional[Dict[str, Callable[[Any], bool]]] = None) -> None:
```

Comment:
I'm not sure if we need to use `Optional` and `None` here. Isn't an "empty" validation `{}` equally good?

Reply:
I wanted to avoid a [mutable default parameter](https://web.archive.org/web/20200221224620id_/http://effbot.org/zone/default-values.htm).

Reply:
Oh, I see. We use mutable default parameters quite often where it's pretty clear that the parameter isn't modified. Whether it would still be better to avoid this potential risk might be a topic for a separate discussion.